tests/OmniSharp.Roslyn.CSharp.Tests/TypeLookupFacts.cs

```diff
@@ -499,7 +499,7 @@ public class TestClass
 }";
             var response = await GetTypeLookUpResponse(content);
             var expected =
-            @"This sample shows how to call the TestClass.GetZero method.
```

Comment:
why did we eliminate these spaces?

Reply:
That doesn't matter because the spaces were not a part of the string anyways. Just left-aligned it